regional-pizza.md

```diff
@@ -0,0 +1,37 @@
+# Regional Pizza
+Welcome to the Regional Pizza section of our repo! 🍕📚
+
+We created this repo into different sections to so we can learn more about each other. If your hometown or country has a special pizza recipe or fact and want to share it, feel free to contribute to this repository and help us expand our knowledge of pizza! 
+
+## Sections
+- [Pizza History](#pizza-history): The origins and evolution of pizza across different cultures and time periods.
```

Comment:
Since there's already a section totally devoted to history, I don't think we need this. I think we can simplify this section. I'm not sure we need the section breakdown here. It might be more useful just to have an example.

Reply:
Sure @BekahHW! 😊 How about this:

"In 1966, two Chicago taxicab company owners started Gino’s East on Superior Street, a single block from the Magnificent Mile. Today, it’s the one Gino’s East location where customers can still add their signatures to the walls. You’ll also find a Gino’s East in the River North Neighborhood. Corn oil adds to the flavor of Gino’s distinctive deep-dish crust. Cooks make sausage pizzas using a full disc of fennel-spiced sausage. This ensures that every bite is a meaty one."
It describes the birth of the Chicago-deep dish pizza.

Reply:
@CBID2 this is great, but I think history should go in the `pizza-facts` section.

In this section, the example would be something like this:

## Ohio Valley Pizza 
Ohio Valley pizza, also known as Steubenville-style pizza or cold-topping pizza, is a unique style of pizza that originates from the Ohio Valley region, particularly Steubenville, Ohio. Here are some key characteristics of Ohio Valley pizza:

- Baked Crust First: Unlike traditional pizzas where toppings are added before baking, Ohio Valley pizza is first baked with just the crust and tomato sauce.
- Cold Toppings: After the crust and sauce are baked, cold toppings like cheese, pepperoni, and other desired ingredients are added on top. The heat from the crust slightly melts the cheese, but the toppings largely remain cold or at room temperature.
- Rectangular Shape: Typically rectangular in shape and often sold by the slice.
- Crispy Crust: The crust is usually chewy and crispy, providing a nice contrast to the cold toppings.
- Served Immediately: Due to its unique preparation method, Ohio Valley pizza is best enjoyed immediately after it's made to experience the contrast of hot crust and cold toppings.

This style of pizza is beloved by many in the Ohio Valley region and has a dedicated following. It offers a different pizza-eating experience, with the juxtaposition of hot and cold elements in every bite.